src/labels/CustomLabels.labels

```diff
@@ -3773,6 +3773,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>bdiFMUIDatatableMapsTo</shortDescription>
         <value>Maps To</value>
     </labels>
+    <labels>
+        <fullName>bdiFMUIDeleteFieldMappingPrompt</fullName>
```

Comment:
@voduyemi - I think this label is one you will need for the Advanced Mapping story you are working on